OVERVIEW.md

```diff
@@ -0,0 +1,123 @@
+# Overview
+
+The `hydra-torch` repo serves three purposes.
```

Comment:
I think the intro needs some work:

maybe something along the lines of:

---
The audience for this library are Hydra users using PyTorch and related libraries (torchvision, ...)
The goals of this repository are:
1. Provide a maintained and tested implementation of config classes that can be used instantiate and configure various classes from the supported projects.
2. Provide examples and tutorials demonstrating best practices for using Hydra to configure PyTorch deep learning applications
3. Showcase a recommended approach for creating other similar configuration packages for other libraries
---